newsapi.py

```diff
@@ -0,0 +1,30 @@
+import click
+import json
+import urllib3
+import requests
+
+newsapikey = '98c49d6124d547ed8e42a05b2915f26a'
```

Comment:
It is generally not a good idea to have API keys within the code base. You can go and read about `.env` files. These files generally help securely store all your secret keys. So instead of using the actual key, you use a reference to .env variable where the key is stored. Read about them and see if maybe you can incorporate that idea here. This is solely for your learning.